pages/Javascript-API.md

```diff
@@ -56,13 +89,69 @@ The properties of the feature at the current cursor location may be accessed thr
 
 This is accomplished by assigning a unique color to each feature onscreen and rendering the scene to an offscreen buffer. When queried, the `scene.selection` object checks the offscreen render at the current cursor position, and identifies the feature by its color.
 
-#### `requestRedraw()`
-Requests an update to the drawn map. If `animated: true` is set, this happens once per frame automatically.
-
 #### `getActiveCamera()`
+Returns the active camera.
 
-#### `setActiveCamera()`
+#### `requestRedraw()`
+Requests an update to the drawn map. If `animated: true` is set, this happens once per frame automatically.
 
 #### `reload(scene_url)`
+Loads the specified scene by url and rebuilds the geometry.
 
 #### `rebuild()`
+Rebuilds the current scene from scratch.
+
+#### `screenshot()`
+This queues a screenshot request, returning a Promise that fulfills when the screenshot is available. (The screenshot must be "queued" because it cannot be captured immediately: we must ensure that the GL buffer is finished drawing, and then must capture the buffer contents just after rendering, before it is cleared by other operations.)
+
+The promise resolves with an object containing two properties:
+
+- url: a data URL of the Canvas contents, suitable for loading into an <img> or opening in a new tab/window
+- blob: a Blob of type image/png, suitable for saving to a file, either manually or with a third-party library such as FileSaver.js
+
+```javascript
+scene.screenshot().then(function(screenshot) {
```

Comment:
an even simpler example without 3rd party dependencies is:

`scene.screenshot().then(function(screenshot) { window.open(screenshot.url); });`

(opens screenshot in a new window)